.maestro/flows/001_002_add_user_direct_test.yaml

```diff
@@ -70,112 +77,265 @@ onFlowComplete:
 - inputText: "+1"
 - tapOn: "Phone number"
 - inputText: "23"
-- hideKeyboard
+- runFlow:
+    when:
+      notVisible: "Save"
+    commands:
+      - hideKeyboard
 - tapOn: "Save"
 - assertVisible: "Invalid"
-- tapOn: "Phone number"
 - eraseText
 - inputText: "21255543268"
-- hideKeyboard
+- runFlow:
+    when:
+      notVisible: "Save"
+    commands:
+      - hideKeyboard
 - tapOn: "Save"
 - assertVisible: "Invalid"
 - eraseText
 - inputText: "2125554326"
 - tapOn: "Email"
-- inputText: "Testuser@gm"
-- hideKeyboard
-- scrollUntilVisible:
-    element:
-      text: "Save" # or any other selector
-    direction: UP
-    timeout: 1000
+- inputText: "ParentAuser@gm"
+- runFlow:
+    when:
+      notVisible: "Save"
+    commands:
+      - hideKeyboard
 - tapOn: "Save"
 - assertVisible: "Enter valid email address."
 - eraseText
-- inputText: "Testuser@gmail.com"
-- assertVisible: "Testuser@gmail.com"
-- hideKeyboard
+- inputText: "ParentAuser@gmail.com"
+- runFlow:
+    when:
+      notVisible: "Save"
+    commands:
+      - hideKeyboard
+- tapOn: "Save"
+- assertVisible:
+    id: "app_title"
+    text: "ParentA User"
+# Add Family member - as new person
+- tapOn:
+    id: "floating_action_button"   # Edit button
+- assertVisible:
+    id: "app_title"
+    text: "Edit person"
+- tapOn: "Family member"
+- tapOn: "Add person"
+- tapOn: "First names*"
+- inputText: "Child"
+- tapOn: "Last name*"
+- inputText: "User"
+- tapOn: "Gender*"
+- tapOn: "Female"
+- assertVisible:
+    id: "app_title"
+    text: "Add person"
+- tapOn: "Save"
+- assertVisible: "Child User"
+- tapOn: "Save"
+- assertVisible:
+    id: "app_title"
+    text: "ParentA User"
+- tapOn: "Child User"
+- assertVisible:
+    id: "app_title"
+    text: "Child User"
+- assertVisible: "Student"
+- assertVisible: "ParentA User"
+- tapOn: "People"
+- assertVisible:
+    id: "app_title"
+    text: "People"
+- tapOn:
+    id: "floating_action_button"   #  +Person button
+- assertVisible:
+    id: "app_title"
+    text: "Add person"
+- tapOn: "First names*"
+- inputText: "Test"
+- tapOn: "Last name*"
+- inputText: "User"
+- tapOn: "Gender*"
+- tapOn: "Female"
+- tapOn:
+    id: "role"
+- tapOn: "Student"
 - tapOn: "Save"
 - assertVisible:
     id: "app_title"
     text: "Test User"
+
+# Create account for Parent user  -- only username and password fields available for parent user
+- tapOn: "People"
+- tapOn: "ParentA User"
+- tapOn: "Create account"
+- assertVisible:
+    id: "app_title"
+    text: "Create account"
+#- tapOn: "Save"
+#- assertVisible: "Require field"  # username is mandatory field
+- tapOn: "Username"
+- inputText: "pa"
+- tapOn: "Save"
+- assertVisible: "Username must be at least 3 characters"
```

Comment:
@poojaustad why is this validation of password length a change in the badge pull request?

Reply:
The password length validation was not working on this screen earlier. Anugraha fixed it while working on the Create Account screen, so I added a test to cover that change.